plonky2/src/util/timing.rs

```diff
@@ -135,16 +171,26 @@ impl TimingTree {
 #[macro_export]
 macro_rules! timed {
     ($timing_tree:expr, $level:expr, $ctx:expr, $exp:expr) => {{
+        #[cfg(feature = "timing")]
```

Comment:
Could we keep these calls and stub them out, so that we don't need to suppress unused warnings (except in the stub methods) or not-mutated warnings?

Reply:
sure

Reply:
the warnings still occur if I stub out the methods. I actually have to mutate in the methods for the warning to go away.